src/config.rs

```diff
@@ -0,0 +1,131 @@
+use anyhow::Result;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+
+/// Represents a single conversion configuration.
+#[derive(Deserialize, Debug)]
+struct Conversion {
```

Comment:
I think we need the ability to specify a destination Rerun entity here as well. Maybe with a behavior that if not-specified we default entity_path = topic.

Reply:
True, I hadn't thought about that, good catch